bluebrain/repo-bluebrain/packages/py-archngv/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyArchngv(PythonPackage):
+    """Building workflow and circuit API for Neuro-Glia-Vascular circuits."""
+
+    homepage = "https://bbpgitlab.epfl.ch/nse/ArchNGV"
+    url      = "git@bbpgitlab.epfl.ch:nse/ArchNGV.git"
+
+    version("develop", branch="main")
+    version("2.0.0", tag="ArchNGV-v2.0.0")
+
+    depends_on("py-setuptools@42:", type="build")
+
+    depends_on("py-numpy@1.19.5:", type=("build", "run"))
+    depends_on("py-scipy@1.5.0:", type=("build", "run"))
+    depends_on("py-h5py@3.1.0:", type=("build", "run"))
+    depends_on("py-libsonata@0.1.8:0.99", type=("build", "run"))
+    depends_on("py-bluepysnap@0.13:0.99", type=("build", "run"))
+    depends_on("py-cached-property@1.5:", type=("build", "run"))
+    depends_on("py-voxcell@3.0.0:", type=("build", "run"))
+    depends_on("py-vascpy@0.1.0:", type=("build", "run"))
+
+    depends_on("py-ngv-ctools@1.0:", type=("build", "run"))
+    depends_on("spatial-index@0.4.2:0.99", type=("build", "run"))
+    depends_on("py-bluepy-configfile@0.1.11:", type=("build", "run"))
+    depends_on("py-click@7.0:7.99", type=("build", "run"))
+    depends_on("py-numpy-stl@2.10:2.99", type=("build", "run"))
```

Comment:
It should match the deps in setup.py
```
numpy-stl>=2.10,<2.16.0
```

Reply:
```
depends_on("py-numpy-stl@2.10:2.15.1", type=("build", "run"))
```
This should do it, right? `2.15.1` is the last release available before `2.16.0`

Reply:
Yes it will work because 2.16.0 is already released, and 2.15.2 won't exist.
In general, to include all the 2.15.x you could write:
```
depends_on("py-numpy-stl@2.10:2.15.999", type=("build", "run"))
```
If I'm not wrong it can be also expressed in the new notation introduced with spack 0.17:
```
depends_on("py-numpy-stl@2.10:2.15", type=("build", "run"))
```